test-clients/src/main/java/com/hedera/services/bdd/suites/regression/SteadyStateThrottlingCheck.java

```diff
@@ -106,22 +113,25 @@ public static void main(String... args) {
 	@Override
 	protected List<HapiApiSpec> getSpecsInSuite() {
 		return List.of(
-				new HapiApiSpec[] {
-						setArtificialLimits(),
+				new HapiApiSpec[]{
+						setArtificialLimits(artificialLimits),
 						checkTps("Xfers", expectedXferTps, xferOps()),
 						checkTps("FungibleMints", expectedFungibleMintTps, fungibleMintOps()),
 						checkTps("ContractCalls", expectedContractCallTps, scCallOps()),
 						checkTps("CryptoCreates", expectedCryptoCreateTps, cryptoCreateOps()),
-//						checkTps("NftMints", expectedNftMintTps, nftMintOps()),
+						checkTps("NftMints", expectedNftMintTps, nftMintOps()),
```

Comment:
This was commented out, did we mean to uncomment it?

Reply:
I made this by mistake. Fixed.